ddl.dic

```diff
@@ -1002,6 +1002,26 @@ save_enumeration_default.index
 
 save_
 
+save_enumeration_default.source_id
+
+    _definition.id                '_enumeration_default.source_id'
+    _definition.update            2023-06-02
+    _description.text
+;
+    An identifier which corresponds to an entry in the ENUMERATION_SOURCE
+    category loop. Used to provide a reference to the original source of
+    a specific enumeration value.
+;
+    _name.category_id             enumeration_default
+    _name.object_id               source_id
```

Comment:
Aren't we now using `ENUMERATION_DEFAULTS`? (`S` on the end)

Reply:
That's in #404, and not merged yet.

Reply:
Sorry, #399